stocks/models.py

```diff
@@ -275,8 +274,6 @@ def value_on(self, date=None):
         Returns the current value of the stock configuration
         """
         value = self.stock.latest_quote(date).value * self.quantity
-        if math.isnan(value):
-            raise Exception("Not able to calculate value")
```

Comment:
Keep this!

Reply:
why? I can't figure out how it would possible happen (see comment above)